libbeat/docs/command-reference.asciidoc

```diff
@@ -20,12 +20,23 @@
 :modules-command-short-desc: Manages configured modules
 :run-command-short-desc: Runs {beatname_uc}. This command is used by default if you start {beatname_uc} without specifying a command
 
+ifndef::deprecate_dashboard_loading[]
+
 ifdef::has_ml_jobs[]
 :setup-command-short-desc: Sets up the initial environment, including the index template, Kibana dashboards (when available), and machine learning jobs (when available)
 endif::[]
 
 ifndef::has_ml_jobs[]
-:setup-command-short-desc: Sets up the initial environment, including the index template, Kibana dashboards (when available)
+:setup-command-short-desc: Sets up the initial environment, including the index template and Kibana dashboards (when available)
+endif::[]
+
+endif::[]
```

Comment:
Could this be combined to a larger if statement? Also below you frequently have multiple if statements per paragraph which could be combined to a larger one. Is that on purpose?

Reply:
The reason is that using larger blocks here would mean more duplication, as in between the blocks we have parts valid for both cases, including even headlines. 

If you prefer larger code blocks with more duplication I'll change it.

Reply:
Definitively prefer less duplication. All good.